### PR TITLE
ENG-608 vizualization issues, removed components

### PR DESCRIPTION
**Pull request by rimselis**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **gidesan**

If they are not already assigned, you can assign the PR to them by writing `/assign @gidesan` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/state/loading/types.js
Comment (ichalagashvili) [inline src/state/loading/types.js:1]: remove this line please :) not needed anymore

File: src/state/loading/actions.js
Comment (ichalagashvili) [inline src/state/loading/actions.js:17]: was it super necessary to add new function other than using `toggleLoading` ?

**Review by ichalagashvili — CHANGES_REQUESTED** [review]

File: src/state/loading/actions.js
Comment (rimselis) [reply src/state/loading/actions.js:17]: There are few places that call the same async fetch at similar time - that makes initial loading value toggle twice or more, making toggle unreliable

**Review by ichalagashvili — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/737#pullrequestreview-439768155" title="Approved">ichalagashvili</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [ichalagashvili]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
